app/controllers/search_controller.rb

```diff
@@ -1,5 +1,6 @@
 class SearchController < LoggedAreaController
   before_action :search_for_dataset, only: [:search]
+  before_action :toggle_beta_message
```

Comment:
already present in `ApplicationController` and `LoggedAreaController` inherits from it, so this `before_action` is not need I think.

Reply:
👍